### PR TITLE
Remove garage.tf.regressors

### PR DESCRIPTION
**Pull request by yeukfu**

### CODE REVIEW

File: src/garage/tf/baselines/gaussian_cnn_baseline.py [hunk 1]
```diff
@@ -7,10 +7,10 @@
 from garage import make_optimizer
 from garage.misc.tensor_utils import normalize_pixel_batch
 from garage.np.baselines.baseline import Baseline
+from garage.tf.baselines.gaussian_cnn_regressor_model import (
```
Comment (ryanjulian) [inline src/garage/tf/baselines/gaussian_cnn_baseline.py:10]: can you just inline that class into this file? make it a `_PrivateClass`

File: src/garage/tf/baselines/gaussian_mlp_baseline.py [hunk 2]
```diff
@@ -5,10 +5,10 @@
 
 from garage import make_optimizer
 from garage.np.baselines import Baseline
+from garage.tf.baselines.gaussian_mlp_regressor_model import (
```
Comment (ryanjulian) [inline src/garage/tf/baselines/gaussian_mlp_baseline.py:8]: similarly -- can you inline this class and rename it `_GaussianMLPBaselineModel`?

File: src/garage/tf/baselines/gaussian_cnn_baseline.py (hunk 1 above)
Comment (ryanjulian) [inline src/garage/tf/baselines/gaussian_cnn_baseline.py:10]: can you rename this module/class `GaussianCNNRegressorModel`?

File: src/garage/tf/baselines/gaussian_mlp_baseline.py (hunk 2 above)
Comment (ryanjulian) [inline src/garage/tf/baselines/gaussian_mlp_baseline.py:8]: Please rename the module/package according to `GaussianMLPBaselineModel`

File: src/garage/tf/baselines/gaussian_cnn_baseline.py (hunk 1 above)
Comment (yeukfu) [reply src/garage/tf/baselines/gaussian_cnn_baseline.py]: Renamed

**Comment by ryanjulian** [comment]
@Mergifyio rebase

**Comment by ryanjulian** [comment]
@Mergifyio rebase

**Pull request merged**